ksql-common/src/test/java/io/confluent/ksql/GenericRowTest.java

```diff
@@ -122,4 +128,51 @@ public void shouldPrintStructRowCorrectly() {
 
   }
 
-}
\ No newline at end of file
+  @Test
+  public void testToString() {
+      LinkedList<Object> linkedList = new LinkedList<>();
+      GenericRow genericRow = new GenericRow(linkedList);
+      linkedList.addLast( null);
+
+      assertEquals("[ null ]", genericRow.toString());
+      assertEquals(1, linkedList.size());
+  }
+
+  @Test
+  public void testGetColumnsWhenNoneAreSet() {
+      List<Object> linkedList = new LinkedList<>();
+      GenericRow genericRow = new GenericRow(linkedList);
+
+      assertEquals(0, linkedList.size());
+      assertEquals(0, genericRow.getColumns().size());
+  }
+
+  @Test
+  public void testEquals(){
+
+      List<Object> columnListWithEmptyObject = new ArrayList<>(1);
+      columnListWithEmptyObject.add(new Object());
+
+      List<Object> columnListWithString = new ArrayList<>(1);
+      columnListWithString.add("nr");
+
+      new EqualsTester().
+              addEqualityGroup(
+                      new GenericRow(),
+                      new GenericRow()
+              ).
+              addEqualityGroup(
+                      new GenericRow(new Object())
+              ).
+              addEqualityGroup(
+                      new GenericRow(columnListWithEmptyObject),
+                      new GenericRow(columnListWithEmptyObject)
```

Comment:
not necessary to have this since we already have the one below (we get coverage knowing that equals covers the inner list, we don't need to cover it twice)